generators/linter/templates/grunt/mochaTest.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  test: {
```

Comment:
@neerajsachan Do we need a `target` here? @riteshe63 Can you check if this is compatible with the mocha.opts file?
@neerajsachan the path is not right! We need to run tests based on the folder structure. So it can be `./test/unit/**/*.js` or something else. @riteshe63 Please advice!